bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -0,0 +1,205 @@
+
+    tags = [] if tags is None else tags
+
+    # Initialize job node with its tags
+    grb_name = workflow.cp.get('workflow', 'trigger-name')
+    extra_tags = ['GRB'+grb_name]
+    extra_tags += [snr_type]
+    if ifo is not None:
+        extra_tags += [ifo]
+    node = PlotExecutable(workflow.cp, 'pygrb_plot_snr_timeseries',
+                          ifos=workflow.ifos, out_dir=out_dir,
+                          tags=tags+extra_tags).create_node()
+    trig_file_path = workflow.cp.get('workflow', 'trig-file')
+    node.add_opt('--trig-file', trig_file_path)
+    # Veto directory and segments directory
+    node.add_opt('--veto-directory', workflow.cp.get('workflow', 'veto-directory'))
+    node.add_opt('--segment-dir', workflow.cp.get('workflow', 'segment-dir'))
+    # Other shared tuning values
+    for opt in ['chisq-index', 'chisq-nhigh', 'null-snr-threshold',
+                'veto-category', 'snr-threshold', 'newsnr-threshold',
+                'sngl-snr-threshold', 'null-grad-thresh', 'null-grad-val']:
+        node.add_opt('--'+opt, workflow.cp.get('workflow', opt))
+    node.new_output_file_opt(workflow.analysis_time, '.png',
+                             '--output-file', tags=extra_tags)
+    # Quantity to be displayed on the y-axis of the plot
+    node.add_opt('--y-variable', snr_type)
+    if ifo is not None:
+        node.add_opt('--ifo', ifo)
+    reset_central_time = shift_time - central_time
+    # Horizontal axis range the = prevents errors with negative times
+    x_lims = str(-5.+reset_central_time)+','+str(reset_central_time+5.)
+    node.add_opt('--x-lims='+x_lims)
+    # Plot title
+    if ifo is not None:
+        title_str = "'%s SNR at %.3f (s)'" %(ifo, central_time)
+        node.add_opt('--central-time', central_time)
+    else:
+        title_str = "'%s SNR at %.3f (s)'" %(snr_type.capitalize(), central_time)
+        node.add_opt('--central-time', central_time)
+    node.add_opt('--plot-title', title_str)
+
+    # Add job node to workflow
+    workflow += node
+
+    return node.output_files
+
+
+# =============================================================================
+# Main script starts here
+# =============================================================================
+parser = argparse.ArgumentParser(description=__doc__[1:])
+parser.add_argument('--version', action='version', version=pycbc.version.git_verbose_msg)
+parser.add_argument("-v", "--verbose", default=False, action="store_true",
+                    help="Verbose output")
+parser.add_argument('--followups-file',
+                    help="HDF format file containing trigger/injections to follow up")
+#parser.add_argument('--instrument', help="Name of interferometer e.g. H1")
+parser.add_argument('--wiki-file',
+                    help="Name of file to save wiki-formatted table in")
+parser.add_argument('--output-map')
+parser.add_argument('--transformation-catalog')
```

Comment:
There is no longer a separate transformation catalog file, so this option should be removed.